packages/components/src/mobile/image/index.native.js

```diff
@@ -225,7 +225,7 @@ const ImageComponent = ( {
 							}
 							openMediaOptions={ openMediaOptions }
 							url={ url }
-							pickerOptions={ mediaPickerOptions }
+							onClearMedia={ onClearMedia }
```

Comment:
Just a headsup that the Gallery block uses Image blocks under the hood AFAIK, so any changes to the `ImageEditingButton` here could affect the Gallery block.

Reply:
Indeed. I will check the Gallery block as well to see how it's utilizing the `ImageEditingButton` and ensure that there aren't any regressions. 😄